src/store/actions/healthcheck.actions.js

```diff
@@ -0,0 +1,102 @@
+import R from 'ramda';
+import client from '../api';
+import {
+    CLEAR_HEALTHCHECK_DETAILS,
+    FETCH_HEALTHCHECK_LIST_START,
+    FETCH_HEALTHCHECK_LIST_SUCCESS,
+    FETCH_HEALTHCHECK_START,
+    FETCH_HEALTHCHECK_SUCCESS,
+    DISCARD_PAGINATION,
+    SET_PAGINATION_PAGE,
+} from '../constants';
+import {
+    openResponseModal,
+} from './index';
+
+export const objectToArray = obj => {
+    const list = Object.keys(obj).reduce((acc, key) => {
```

Comment:
why don't you use ramda for it?

Reply:
thanks @nik-kor ! just forgot about `R.keys`

Reply:
I meant for the whole transformation...
I saw your comment